examples/mlflow_deployment/steps/predictor/predictor_step.py

```diff
@@ -26,6 +26,6 @@ def predictor(
     """Run a inference request against a prediction service."""
     service.start(timeout=10)  # should be a NOP if already started
     prediction = service.predict(data)
-    prediction = prediction.argmax(axis=-1)
+    prediction = np.apply_along_axis(lambda x: x.argmax(), -1, prediction)
```

Comment:
Doesn't this do the same as the old code?

Reply:
Yes it's the same, i changed this just as sort of debugging something but forgot to revert it back

Reply:
Then I would prefer the old syntax